src/ngraph/pass/opset0_downgrade.cpp

```diff
@@ -83,6 +84,62 @@ bool pass::Opset0Downgrade::run_on_node(shared_ptr<Node> node)
 #endif
     switch (get_typeid(node))
     {
+    case OP_TYPEID::Convolution:
+    {
+        auto tmp = as_type_ptr<op::v1::Convolution>(node);
+        const auto data_arg = node->input(0).get_source_output();
+        const auto filters_arg = node->input(1).get_source_output();
+        const size_t num_spatial_dims = data_arg.get_shape().size() - 2;
```

Comment:
What if shape will be dynamic?
You should read `data_arg` rank and check if it is dynamic or not IMO.